src/unitxt/metrics.py

```diff
@@ -12,6 +13,7 @@
 import evaluate
 import numpy
 import numpy as np
+from fuzzywuzzy import fuzz
```

Comment:
Please import this where used and not in main scope - so if people don't use the metric, it will not fail.